lizard_auth_client/models.py

```diff
@@ -82,6 +83,14 @@ class Meta:
     def __str__(self):
         return self.name
 
+    @property
+    def uuid(self):
```

Comment:
Is het de bedoeling dat alleen deze property gebruikt wordt en niet `unique_id`?
Dan moet er misschien een korte opmerking bij unique_id geplaatst worden.

Eén van de twee zal wel de baas zijn en de ander een helper (gok ik).